_posts/en/newsletters/2019-06-06-newsletter.md

```diff
@@ -0,0 +1,218 @@
+an unknown amount of time creates a risk that the customer will go away."  For
+example, by the time a deposit clears at an exchange, the customer may no
+longer be interested in making the trade that would've earned the exchange a
+commission.  Additionally, Bitrefill's experience with LN is that LN's improved
+invoicing eliminates a number of different payment errors seen with onchain
+bitcoin payments, including overpayments, underpayments, stuck transactions,
+copy/paste errors, and other problems.  Receiving payments over LN also
+eliminates the need to consolidate UTXOs and reduces the need to rotate money
+between hot and cold wallets.  Eliminating all of these problems has the potential to significantly reduce customer support and backend expenses.
+
+    In a particularly interesting section of his talk, Kotliar shows how perhaps as
+much as 70% of the onchain payments on the network today are users moving money
+from one exchange to another exchange (or even between different users of the
+same exchange).  If all that activity could be moved offchain using LN
+payments, exchanges and their users could save a considerable amount of money.
+
+    Kotliar concluded his talk with a few short segments.  He described what
+software and services Bitrefill sees LN users using today and what he
+expects them to be using in the near future.  He then explained two of
+Bitrefill's services for LN users (including businesses), [Thor][] and [Thor
+Turbo][].  Finally, he briefly described several planned improvements to LN:
+reusable addresses (see [Newsletter #30][]), splicing in and out (see
+[#22][Newsletter #22]), and larger channels (also [#22][Newsletter
+#22]).
+
+    Overall, Kotliar made a compelling case that LN's faster speed, lower fees,
+and improved invoicing means businesses that expect to remain competitive
+serving Bitcoin customers in the near future should start working on
+implementing LN support today.
+
+- **COSHV proposal replaced:** the author of the COSHV proposal we described
+  [last week][news48 coshv] has replaced it with a [similar
+  proposal][alt-coshv] (under a different name).  The new proposal checks more
+  than just the hash of a transaction's outputs, now it includes the transaction's
+  version number, number of inputs, sequence numbers, and locktime.  This
+  change eliminates concerns related to transaction malleability that would've
+  affected using the opcode with some types of payment channels (such as those
+  used by LN) and possibly other contract protocols.  Additionally, by hashing the
+  number of inputs allowed in the transaction, the original proposal's
+  restriction on just one input is lifted (however, the proposal notes that only
+  allowing a single input is still recommended to avoid unwanted interactions).
+
+    Except for the new name, the changes don't affect the summary of COSHV we
+    wrote last week.  However, the changes do affect some additional material about
```

Comment:
I did (3).  I don't know what "tween" means in (2), but it also seems predicated on removing this sentence, so I went with that.  If I'm not around, please feel free to add a note (or not).

As for editing last week's newsletter, I wouldn't feel comfortable making a substantial edit like that without leaving a note about the edit, which I think would kinda defeat the purpose.

Reply:
I figured out "tween" probably means "tweet".  I think tweeting that initially is unnecessary, but that it'd be good for any one of us to tweet something like that in reply to anyone asking about the previously-announced text.

Reply:
Yes s/tween/tweet/

Reply:
> I wouldn't feel comfortable making a substantial edit like that without leaving a note about the edit, which I think would kinda defeat the purpose.

I feel comfortable making edits to old newsletters on the website that make them more accurate/clearer for readers, since I see them as a more permanent resource. We can discuss more over the IRC.